pandas/tests/util/test_util.py

```diff
@@ -482,3 +483,23 @@ def test_make_signature():
     assert sig == (['old_arg_name', 'new_arg_name',
                     'mapping=None', 'stacklevel=2'],
                    ['old_arg_name', 'new_arg_name', 'mapping', 'stacklevel'])
+
+
+def test__safe_import():
+    from _pytest.monkeypatch import MonkeyPatch
```

Comment:
What is _pytest? On mobile and can’t check easily.

Reply:
That reflects the package structure of pytest. However, I think this can be re-written another way as I'm reading more on how pytest plugins work. Let me re-push

https://holgerkrekel.net/2009/03/03/monkeypatching-in-unit-tests-done-right/